### PR TITLE
LPS-143474 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-143474

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0"?>
-
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Tim-Cao) [inline .project:1]: Wrong modification

**Comment by Tim-Cao** [comment]
Hi @ozysouza, I picked up some commits in https://github.com/liferay-echo/liferay-portal/pull/6740. Could you send  anthor follow up PR?

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 2]
```diff
@@ -1,5 +1,11 @@
 definition {
 
+	macro checkPage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:366]: selectPageItem

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 3]
```diff
@@ -483,6 +483,24 @@ definition {
 		}
 	}
 
+	macro view {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:618]: viewPageItemPresent

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 4]
```diff
@@ -596,6 +596,20 @@ definition {
 			locator1 = "SitePages#LIST_GROUP_ITEM_CHECKBOX");
 	}
 
+	macro viewNoPage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:688]: viewPageItemNotPresent

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update according to comments? Thanks

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro [hunk 5]
```diff
@@ -258,6 +258,19 @@ definition {
 		return "${authToken}";
 	}
 
+	macro getFriendlyURL {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro:261]: Move to PagesAdmin.macro

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 6]
```diff
@@ -369,6 +369,31 @@ definition {
 		PortletEntry.save();
 	}
 
+	macro deleteLogoPG {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:436]: deleteLogo

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 7]
```diff
@@ -299,6 +299,20 @@ definition {
 		WaitForSPARefresh();
 	}
 
+	macro changeLayoutLogo {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:352]: changeLogo

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 8]
```diff
@@ -299,6 +299,34 @@ definition {
 		WaitForSPARefresh();
 	}
 
+	macro changeCurrentTheme {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:324]: Duplicate with Site.changeCurrentTheme, Could you migrate Site.changeCurrentTheme to PagesAdmin?

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 9]
```diff
@@ -622,11 +622,66 @@ definition {
 			value1 = "${status}");
 	}
 
+	macro viewPageTemplate {
+		Navigator.openSiteURL(siteName = "Site Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:729]: Remove this line to make this macro reusable

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 10]
```diff
@@ -622,11 +622,66 @@ definition {
...
+
 	macro viewPageType {
 		AssertElementPresent(
 			key_pageName = "${pageName}",
 			key_pageType = "${pageType}",
 			locator1 = "SitePages#LIST_GROUP_ITEM_PAGE_TYPE");
 	}
 
+	macro viewPanelPortletListCP {
+		ProductMenuHelper.openProductMenu();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:760]: Redundant step

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 11]
```diff
@@ -596,6 +596,52 @@ definition {
...
+		MenuItem.click(menuItem = "Configure");
+
+		Navigator.gotoNavTab(navTab = "Look and Feel");
+
+		AssertTextPresent(
+			locator1 = "SitePagesEditPage#VIEW_THEME",
+			value1 = "${themeName}");
+	}
+
+	macro viewCurrentThemeSettings {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:660]: macro viewWidgetPageCurrentTheme

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 12]
```diff
@@ -596,6 +596,52 @@ definition {
 			locator1 = "SitePages#LIST_GROUP_ITEM_CHECKBOX");
 	}
 
+	macro viewCurrentTheme {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:642]: Integrate with viewWidgetPageCurrentTheme

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro [hunk 13]
```diff
@@ -562,6 +562,49 @@ definition {
...
 
+	@summary = "Delete all pages except the Welcome and Search Page"
+	macro tearDownCP {
+		var siteName = TestCase.getSiteName(siteName = "${siteName}");
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Site Builder",
+			portlet = "Pages");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pagesadmin/PagesAdmin.macro:573]: `SitePages.openPagesAdmin(siteURLKey = "guest");`

File: .project (hunk 1 above)
Comment (ozysouza) [reply .project:1]: Sorry, I'll remove it. I don't know why but every now and then the system makes some modification in this project file and I always remove it, but this time missed it.

**Comment by Tim-Cao** [comment]
Hi @ozysouza, additional PR is here https://github.com/liferay-echo/liferay-portal/pull/6744

**Comment by ozysouza** [comment]
> Hi @ozysouza, I picked up some commits in #6740. Could you send anthor follow up PR?

Hi @Tim-Cao, I sent some commits in [6751](https://github.com/liferay-echo/liferay-portal/pull/6751) would you please take a look at?

I'll send others PR with these remain macros.

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**